test/appsignal/probes/erlang_probe_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule Appsignal.Probes.ErlangProbeTest do
-  use ExUnit.Case
+  use ExUnit.Case, async: false
```

Comment:
`false` is actually the default, so there is no need to specify this: https://hexdocs.pm/ex_unit/ExUnit.Case.html